sc/src/main/webapp/WEB-INF/jsp/tiles/views/hca-landing-page.jsp

```diff
@@ -0,0 +1,46 @@
+<%--
+  Created by IntelliJ IDEA.
+  User: lingyun
+  Date: 25/10/2018
+  Time: 09:14
+  To change this template use File | Settings | File Templates.
+--%>
+
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+
+<div class="media-object">
+    <div class="media-object-section top hide-for-small-only">
+        <div class="thumbnail">
+            <img  alt="Human Cell Atlas" style="max-width: 275px"
+                  src="${pageContext.request.contextPath}/resources/images/logos/human_cell_atlas.png"/>
+        </div>
+    </div>
+    <div class="media-object-section top">
+        <p>
+            "Single cell Expression Atlas provides access to the analysis results of X data sets
```

Comment:
Remove double quotes in this and the next paragraph. We’re not quoting anybody in particular.